wolfssl/wolfcrypt/sp.h

```diff
@@ -276,6 +276,33 @@ int sp_ecc_proj_dbl_point_384(mp_int* pX, mp_int* pY, mp_int* pZ,
 int sp_ecc_map_384(mp_int* pX, mp_int* pY, mp_int* pZ);
 int sp_ecc_uncompress_384(mp_int* xm, int odd, mp_int* ym);
 
+int sp_ecc_mulmod_521(const mp_int* km, const ecc_point* gm, ecc_point* rm,
```

Comment:
Should all of these including above have `WOLFSSL_LOCAL`?

Reply:
Done.